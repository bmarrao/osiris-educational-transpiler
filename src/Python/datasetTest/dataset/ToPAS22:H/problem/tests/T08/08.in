5
8 11
HRbA ZSgr jLLZ mHa Jg t LkhdT a
t jLLZ
mHa jLLZ
mHa a
t LkhdT
mHa HRbA
a jLLZ
jLLZ ZSgr
HRbA t
a Jg
LkhdT HRbA
LkhdT Jg
8 16
o ifNoY LSW Mrz za Y WEUyd wgl
ifNoY wgl
za LSW
WEUyd o
Y LSW
wgl WEUyd
WEUyd ifNoY
Y o
LSW o
Mrz o
ifNoY o
ifNoY za
Mrz wgl
Y Mrz
ifNoY LSW
o za
wgl Y
8 17
Sew Prj Dfa KmxNz EfCUX mn cx Ey
Ey KmxNz
mn Dfa
mn Sew
KmxNz Dfa
EfCUX cx
Ey cx
mn Ey
Ey EfCUX
cx mn
Sew Ey
KmxNz cx
KmxNz mn
KmxNz EfCUX
Prj mn
Sew Prj
Dfa cx
cx Prj
8 9
Xm V WX HcFwN cRn hjCIu oIkk jhd
HcFwN V
oIkk WX
V WX
jhd V
jhd cRn
V Xm
hjCIu Xm
cRn oIkk
WX HcFwN
8 0
cZ I wnMDN VYkE INr Q gWu Xnlo
