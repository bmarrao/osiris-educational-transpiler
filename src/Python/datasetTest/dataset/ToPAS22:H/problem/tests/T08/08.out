Possivel: 6144
Possivel: 1152
Possivel: 384
Possivel: 8448
Possivel: 40320
